do-not-commit/ssrf-injection-requests.py

```diff
@@ -0,0 +1,64 @@
+def test_bad_1():
+    from requests import get
+    from django.shortcuts import render
+
+    def send_to_redis(request):
+        # ruleid: ssrf-injection-requests
+        bucket = request.GET.get("bucket")
+        inner_response = get("http://my.redis.foo/{}".format(bucket), data=3)
+        return render({"response_code": inner_response.status_code})
+
+
+def test_bad_2():
+    from requests import get
+    from django.http import HttpResponse
+
+    def send_to_redis(request):
+        # ruleid: ssrf-injection-requests
+        bucket = request.GET.get("bucket")
+        inner_response = get("http://my.redis.foo/{}".format(bucket), data=3)
```

Comment:
By default, 'requests' calls wait until the connection is closed.
This means a 'requests' call without a timeout will hang the program
if a response is never received. Consider setting a timeout for all
'requests'. ([actice.use-timeout.use-timeout](https://semgrep.dev/r/actice.use-timeout.use-timeout))
#
<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=actice.use-timeout.use-timeout) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>